packages/recomponents/src/components/r-tabs/r-tab.vue

```diff
@@ -27,6 +33,17 @@
                 isActive: this.active,
             };
         },
+        computed: {
+            accessibilityId() {
+                return this.panelId || this._uid; /* eslint-disable-line no-underscore-dangle */
```

Comment:
thanks for moving this eslint line! 2 things:
1. We always put the eslint statement on a line directly above the relevant code, like this:
```
// eslint-disable-next-line no-underscore-dangle
return this.panelId || this._uid;
```

2. I just took a closer look at what was being done here and did a little research. According to Evan You (founder of Vue.js), we definitely should not use `this._uid` as it is an internal undocumented property. See:
https://github.com/vuejs/vue/issues/5886#issuecomment-308625735

Instead, we should be using `shortId.generate()`. `r-checkbox` has a good example of that usage.

Reply:
That little `shortid` package is pretty neat! 😎 
Thanks for letting me know!